test/bigtest/network/factories/eresource.js

```diff
@@ -0,0 +1,21 @@
+import { Factory, faker } from '@bigtest/mirage';
+
+export default Factory.extend({
+  id: () => faker.random.uuid(),
+  class: () => faker.random.arrayElement(['org.olf.kb.Pkg']),
+  name: () => faker.random.words(),
+  customCoverage: () => false,
+
+
+  afterCreate(eresource, server) {
+    eresource.testData.forEach(item => {
```

Comment:
Maybe given the above, we can also change this from `testData`? What's `testData` mean? Normally I'd assume some very generic thing, but here it's literally an array of PCIs. Simply by calling it something else, we'd be better accomplishing the segmentation we're going to need to handle non-package eresources.